src/main/java/org/spongepowered/api/profile/GameProfile.java

```diff
@@ -37,9 +42,28 @@
 public interface GameProfile extends Identifiable, DataSerializable {
 
     /**
-     * Gets the player's last known username.
+     * Creates a {@link GameProfile} from the provided ID and name.
+     *
+     * @param uniqueId The unique ID
+     * @param name The name
+     * @return The created profile
+     */
+    static GameProfile of(UUID uniqueId, String name) {
```

Comment:
If the name is optional, should we consider that the name should be `@Nullable`? @SpongePowered/developers thoughts?